### PR TITLE
Quick start with `frint-cli`, and getting rid of Makefile in examples

### PR DESCRIPTION
**Pull request by fahad19**
## What's done

* Quick start guide with `frint-cli` in repo's main README
* Got rid of `Makefile` in examples, a step forward to support Windows better

### CODE REVIEW

File: examples/kitchensink/package.json [hunk 1]
```diff
@@ -4,7 +4,11 @@
   "description": "Kitchensink app in Frint",
   "main": "index.js",
   "scripts": {
-    "build": "webpack"
+    "bundle:vendors": "webpack --config ./vendors/webpack.config.js",
+    "bundle:apps": "webpack",
+    "build": "npm run bundle:vendors && npm run bundle:apps",
+    "start:server": "",
```
Comment (jackTheRipper) [inline examples/kitchensink/package.json]: missing `live-server ./build --port $(PORT)` :)

**Comment by jackTheRipper** [comment]
@fahad19 : why don't we keep `start` and `start:dev-server` for all the examples?

**Comment by fahad19** [comment]
what's intended for `start:dev-server`?

**Comment by jackTheRipper** [comment]
Ok you fixed it sorry.

**Pull request merged**